### DISCUSSION TITLE
Skip consent modal

### DISCUSSION THREAD

**Discussion by Anjay-3000** [orestbida/cookieconsent#432] [Q&A]
Greetings and thanks for this great tool. I was wondering if it is possible to skip the consent modal and go directly into the settings modal. I haven't found a quick way to do this while browsing the code. Would be grateful for further help.

**Comment by orestbida** [comment]
Although you can, it is not recommended. Unlike the small consent modal, settings modal will cover the entire page if the user is in mobile view.

I need to know the version you're using to give you an example with the code.

**Reply by Anjay-3000** [reply]
Thanks for the quick response and the hint (I will check how much mobile view will be covered). I am using version 2.8.8.

**Comment by glenr8** [comment]
Hi, I also am interested in exactly this, open settings by default. Any update? I am using CookieConsent v3.0.0-beta.1.

**Reply by orestbida** [reply]
Hi @glenr8,

you need to set [`autoShow: false`](https://cookieconsent.orestbida.com/reference/configuration-reference.html#autoshow) and manually call [`showPreferences`](https://cookieconsent.orestbida.com/reference/api-reference.html#showpreferences):

Example:
```javascript

import { run, showPreferences } from 'cookieconsent.esm.js';

run({
    autoShow: false,
    // your config ...
}).then(() => {
    showPreferences();
});
```
